metadata/service/support.yml

```diff
@@ -4,7 +4,7 @@ parameters:
       collectd:
         enabled: false
       heka:
-        enabled: false
+        enabled: true
```

Comment:
You have to have a `meta/heka.yml` if you enable Heka support. For now I think I would focus on Aodh itself, and defer the support of Heka.